op-acceptance-tests/tests/base/leadership_transfer_test.go

```diff
@@ -0,0 +1,139 @@
+package base
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-conductor/consensus"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+type conductorWithInfo struct {
+	stack.Conductor
+	info consensus.ServerInfo
+}
+
+// TestLeadershipTransfer checks if the leadership transfer works correctly on the conductors
+func TestLeadershipTransfer(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewMinimal(t)
+	logger := testlog.Logger(t, log.LevelInfo).With("Test", "TestLeadershipTransfer")
+	tracer := t.Tracer()
+	ctx := t.Ctx()
+	logger.Info("Started L2 RPC connectivity test")
+
+	ctx, span := tracer.Start(ctx, "test chains")
+	defer span.End()
+
+	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
+	defer cancel()
+
+	// Test all L2 chains in the system
+	for _, l2Chain := range sys.L2Networks() {
+		_, span = tracer.Start(ctx, "test chain")
+		defer span.End()
+
+		require.NotEmpty(t, l2Chain.Escape().Conductors(), "no conductors found in the L2 chain")
+
+		membership := dsl.NewConductor(l2Chain.Escape().Conductors()[0]).FetchClusterMembership()
+		if len(l2Chain.Escape().Conductors()) == 0 {
+			t.Skip("no conductor found in the input, skipping leadership transfer test")
```

Comment:
Just confirming, is it NOT true that every chain will have at least one conductor?

Reply:
No no. Every chain has at least one conductor. Just that their devnet envs don't have it declared yet because of netchef's incapability. So to not end up breaking op-acceptor for those ones, I added this section.

PS: addressing the netchef gap here - https://github.com/ethereum-optimism/infrastructure-services/pull/443/files#diff-b1b0558b577a9c66ac02a5f89c15b86f38fe80e962bc4942417a7789dd7d160e

Reply:
Oh! I see - that makes sense.
Should we add a TODO here then?